examples/rbac-remote/client/oidc/admin_user_resources.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client-admin-user
+  namespace: feast-dev
+  labels:
+    app: client-admin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: client-admin
+  template:
+    metadata:
+      labels:
+        app: client-admin
+    spec:
+      containers:
+        - name: client-admin-container
+          image: feastdev/feature-server:latest
+          imagePullPolicy: Always
+          command: ["sleep", "infinity"]
+          env:
+            - name: KEYCLOAK_CLIENT_SECRET
+              value: <KEYCLOAK_CLIENT_SECRET>
```

Comment:
These values in <> must be manually updated before installing the examples, right?
Is this documented in README? (cannot see it)

Reply:
Removed it from env variable putted into feature_store yaml file and added into readme to update it.